### ISSUE TITLE
invalid GTIN 

### ISSUE THREAD

**Issue by yyyakupyazici** [allegro/allegro-api#9012]
I propose new product. But it is not successful. My trace-id: 9156f4843d46099f. What should I do to get ok?

**Comment by MaciejFrackowiak** [comment]
You should propose a product with the correct EAN (GTIN) value - the value "1234387489" is not valid.

**Comment by yyyakupyazici** [comment]
It seems that EAN (GTIN) value is mandatory. Is there any way to send product with my custom EAN (GTIN).

**Comment by MaciejFrackowiak** [comment]
No, you have to provide an EAN {GTIN) which is valid and [registered](https://allegro.pl/dla-sprzedajacych/od-2-stycznia-nie-wystawisz-oferty-z-nieprawidlowym-kodem-ean-gtin-2Y6W7lommuA) by the GS1 Organization. You cannot create its value by yourself.
